src/core/viz/expressions/base.js

```diff
@@ -53,12 +53,14 @@ export default class Base {
         if (!glslCode) {
             return '';
         }
+        
         return `
-        #ifndef DEF_${this._uid}
-        #define DEF_${this._uid}
-        ${glslCode}
-        #endif
-        `;
+            \n
```

Comment:
do we need explicit whitespace with string literals?

Reply:
Without them it does not compile :S

Reply:
the gpu? or webpack?

Reply:
What do you mean by "explicit whitespace"? :confused:

Reply:
Javascript string literals capture whitespace chars (newline, tabs...) so:

```js
console.log(`

hello

david
`);
```

is equivalent to:

```js
console.log(`\nhello\n\ndavid`);
```

Reply:
Yes, of course, but... that change must work xD

Reply:
Without breaklines you end up with something similar to:

```
#endif#ifndef DEF_1
```
And the loader triggers an error. I would have to think a better solution, but if you have any suggestion, is more than welcome :)

Reply:
My point is I think we dont need explicit `\n` we can just insert a newline